CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ project(
   GOOFIT
   VERSION 2.2.3
   LANGUAGES CXX)
-
+SET(CUDA_SEPARABLE_COMPILATION ON)
```

Comment:
This should already be set, I believe. Also, it should be set only for CUDA backend (which I think it already is).

Reply:
you are right, this was already set and this line is not needed